src/zc/recipe/testrunner/tests.py

```diff
@@ -12,57 +12,166 @@
+        output = system(os.path.join(self.tmp, 'bin', 'test') + ' -vv -j2')
+        comp_lines = OUTPUT_COMP.split('\n')
+
+        # Here we check if meaningful outputs have been given. See above.
+        self.assertTrue(OUTPUT_COMP)
+        self.assertTrue(comp_lines)
+        for line in comp_lines:
+            self.assertIn(line, output)
+
+
 def setUp(test):
     zc.buildout.testing.buildoutSetUp(test)
+    zc.buildout.testing.install('six', test)
     zc.buildout.testing.install_develop('zc.recipe.testrunner', test)
     zc.buildout.testing.install_develop('zc.recipe.egg', test)
     zc.buildout.testing.install('zope.testing', test)
     zc.buildout.testing.install('zope.testrunner', test)
     zc.buildout.testing.install('zope.interface', test)
     zc.buildout.testing.install('zope.exceptions', test)
 
+
 def test_suite():
     return unittest.TestSuite((
         doctest.DocFileSuite(
             'README.txt',
-            'bugfixes.txt',
             setUp=setUp, tearDown=zc.buildout.testing.buildoutTearDown,
             checker=zope.testing.renormalizing.RENormalizing(
-                    [zc.buildout.testing.normalize_path,
-                     zc.buildout.testing.normalize_script,
-                     zc.buildout.testing.normalize_egg_py,
-                     zc.buildout.testing.normalize_endings,
-                     (re.compile('#!\S+py\S*'), '#!python'),
-                     (re.compile('\d[.]\d+ seconds'), '0.001 seconds'),
-                     (re.compile('zope.testing-[^-]+-'), 'zope.testing-X-'),
-                     (re.compile('zope.testrunner-[^-]+-'), 'zope.testrunner-X-'),
-                     (re.compile('setuptools-[^-]+-'), 'setuptools-X-'),
-                     (re.compile('distribute-[^-]+-'), 'setuptools-X-'),
-                     (re.compile('zope.interface-[^-]+-'), 'zope.interface-X-'),
-                     (re.compile('zope.exceptions-[^-]+-.*\.egg'), 'zope.exceptions-X-pyN.N.egg'),
-                     #windows happiness for ``extra-paths``:
-                     (re.compile(r'[a-zA-Z]:\\\\usr\\\\local\\\\zope\\\\lib\\\\python'),
-                                '/usr/local/zope/lib/python'),
-                     #windows happiness for ``working-directory``:
-                     (re.compile(r'[a-zA-Z]:\\\\foo\\\\bar'),
-                                '/foo/bar'),
-                     #more windows happiness:
-                     (re.compile(r'eggs\\\\'),
-                                'eggs/'),
-                     (re.compile(r'parts\\\\'),
-                                'parts/'),
-                     ]),
-            ),
+                [zc.buildout.testing.normalize_path,
+                    zc.buildout.testing.normalize_script,
+                    zc.buildout.testing.normalize_egg_py,
+                    zc.buildout.testing.normalize_endings,
+                    (re.compile('#!\S+py\S*'), '#!python'),
+                    (re.compile('\d[.]\d+ seconds'), '0.001 seconds'),
```

Comment:
And here I suggest adding a

```python
                    (re.compile(r'\d[.]\d+ s\b'), '0.000 s'),
```

instead of relying on the `...` in the doctest.